inventories/README.md

```diff
@@ -40,13 +49,15 @@ remove_all_networkd_config: true
 # THIS MUST NOT BE USED WITH skip_recreate_team0_config at the same time or the cluster network config won't be recreated.
 ```
 
-## Basic cluster inventory
+### Demo SEAPATH cluster inventory
+
+If we are SEAPATH newcomer, we can use this example to create a SEAPATH POC cluster. At the opposite of a real SEAPATH cluster, which requires an advance machine with multiple cores and network cards. This example only requires one network interface and can be made with costumer retail machines. It had to be tested by the SEAPATH team with Intel NUC machines.
```

Comment:
> It had to be tested by the SEAPATH team with Intel NUC machines.

I don't think this sentence is needed here

Reply:
@dupremathieu what do you think of this ?

Reply:
sentence removed